Tasks/AzureResourceGroupDeploymentV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -194,5 +194,6 @@
   "loc.messages.DeploymentGroupConfigurationFailedOnVM": "Failure for Virtual Machine '%s': %s",
   "loc.messages.ServicePrincipalRoleAssignmentDetails": "Please make sure the Service Principal with name %s is assigned the right roles for the Resource Group %s. Follow the link for more details: https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal",
   "loc.messages.ServicePrincipalFetchFailed": "Error while fetching Service Principal details: %s",
-  "loc.messages.FindMoreDeploymentDetailsAzurePortal": "Task successfully created an Azure Resource Manager deployment, but the deployment failed. Please see more detailed Azure resource manager deployment logs at: (Please Copy-Paste the link) %s"
+  "loc.messages.FindMoreDeploymentDetailsAzurePortal": "Task successfully created an Azure Resource Manager deployment, but the deployment failed. Please see more detailed Azure resource manager deployment logs at: (Please Copy-Paste the link) %s",
+  "loc.messages.TemplateValidationFailure": "Validation errors were found in the Azure Resource Manager template. %s"
```

Comment:
```suggestion
  "loc.messages.TemplateValidationFailure": "Validation errors were found in the Azure Resource Manager template. This can result in Azure resource manager failing the template deployment. If the deployment fails, please ensure that you fix the validation error and re-run the task. Error: %s"
```

Reply:
Additionally is the a ARM template validation doc which we can link here?

Reply:
Is there such a doc link?